astropy/utils/misc.py

```diff
@@ -327,6 +331,7 @@ def _has_hidden_attribute(filepath):
         return False
 
 
+@deprecated(since="6.0", alternative="'is_path_hidden() from 'extension-helpers'")
```

Comment:
I’m not sure if the function in extension helpers should be considered public API there so I would actually recommend not specifying an alternative.